src/main/kotlin/sliv/tool/parsers/lines_parsers/CSVLinesParser.kt

```diff
@@ -0,0 +1,42 @@
+package sliv.tool.parsers.lines_parsers
+
+import com.fasterxml.jackson.dataformat.csv.CsvMapper
+import com.fasterxml.jackson.dataformat.csv.CsvSchema
+import com.fasterxml.jackson.module.kotlin.KotlinModule
+import sliv.tool.data_structures.landmarks.LineLandmark
+import sliv.tool.data_structures.landmarks.Point
+import sliv.tool.parsers.ParserUtils
+
+class CSVLinesParser : ILinesParser {
```

Comment:
how to use such parser? create for each file? maybe any singleton facade is required?